BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -29,148 +29,157 @@ class SideMenuFilterViewController: UITableViewController {
     @IBOutlet weak var cancelButton: UIButton!
     @IBOutlet weak var contentViewCell: UIView!
     @IBOutlet weak var slideValueLabel: UILabel!
-    
-    var distanceFilter = 0
+
+    var distanceFilter: CGFloat = 0.00
```

Comment:
Was looking at some people dealing with the rounding issue and the round() works with CGFloat, tho I ended up doing it a bit differently so i can reevaluate